BrainPortal/app/views/tool_configs/_by_resource.erb

```diff
@@ -149,30 +149,31 @@
               users_with_access_pb = users.select do |user|
                 !user_can_access_resource[user] || !user_can_access_versus[user] || !tc.can_be_accessed_by?(user)
               end
+              users_with_access = users - users_with_access_pb
             %>
-            <%=
-              array_to_table(users_with_access_pb,
-                :rows        => 1,
-                :table_class => 'simple',
-                :td_class    => 'warning'
-              ) do |user,r,c|
-            %>
+            <% tool_pb, exec_pb = user_can_access_versus, user_can_access_resource %>
+            <% exec_pb, tool_pb = tool_pb, exec_pb if for_tool %>
               <!-- Add information on the tooltip -->
-              <%= html_tool_tip(user.login) do %>
-                <%= "* Execution Server's project<br/>".html_safe      unless
-                  ( for_tool ? user_can_access_versus[user] : user_can_access_resource[user] ) %>
-                <%= "* Tool's project<br/>".html_safe                  unless
-                  ( for_tool ? user_can_access_resource[user] : user_can_access_versus[user] ) %>
-                <%= "* Version configuration's project<br/>".html_safe unless tc.can_be_accessed_by?(user) %>
+            <span style="color:green">
```

Comment:
Please don't hardcode element when there is a helper that can do that.

Use html_colorize()

Reply:
done